connector/pom.xml

```diff
@@ -14,6 +14,18 @@
 
     <dependencies>
         <!-- Compile Scope -->
```

Comment:
Please update `                <artifactId>scala-maven-plugin</artifactId>`
To use   target:jvm-11 (or however its supposed to be called)